src/services/posts/like-post.test.js

```diff
@@ -4,43 +4,92 @@ const likePost = require("./like-post")
 const {fixtures} = require('../../../test/utils')
 
 
-describe('RECORD LIKES on gif', () => {
+describe('RECORD LIKES ', () => {
 	let user
-	let post
 	before(async ()=>{
-		 user = await fixtures.insertUser() 
-		post = await fixtures.insertPost({
-			userId : user.id , 
-			type : 'gif'
-		})
+		user = await fixtures.insertUser() 
 	})
-	it('should throw an error if Gif is already liked', async () => {
-		await fixtures.insertPostLike({
-			userId : user.id,
-			postId: post.id
+	describe('Gif', () => {
+		
+		let post
+		before(async ()=>{
+			user = await fixtures.insertUser() 
```

Comment:
not sure inserting a new user here affects the test?

Reply:
it was an oversight, i have rectified it now